src/sql/src/plan/statement/dml.rs

```diff
@@ -283,9 +285,28 @@ with_options! {
 
 pub fn describe_tail(
     scx: &StatementContext,
-    TailStatement { name, options, .. }: TailStatement<Raw>,
+    TailStatement {
+        relation, options, ..
+    }: TailStatement<Raw>,
 ) -> Result<StatementDesc, anyhow::Error> {
-    let sql_object = scx.resolve_item(name)?;
+    let relation_desc = match relation {
+        TailRelation::Name(name) => scx.resolve_item(name)?.desc()?.clone(),
+        TailRelation::Query(query) => {
+            // ORDER BY isn't supported by TAIL. As a result, LIMIT and OFFSET cease to be
+            // meaningful since users can no longer guarantee the order of results.
```

Comment:
Hmm, why not? Seems like an easy `TopK` to smush onto the end, just like we do for a subquery? No need to do it here, but can you file an issue so it doesn't get lost?

Reply:
Isn't this because it's tail? If you do `TAIL (SELECT * from some_materialized_view)` then as that view changes we get new diffs and order by doesn't mean anything.

Reply:
Oh wait, yeah, this should be fine. Was confused.